README.md

```diff
@@ -19,4 +19,5 @@ For more information, please see the links below.
 
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fcodecov%2Fexample-python.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fcodecov%2Fexample-python?ref=badge_large)
 
-test
+test2
```

Comment:
As a general note, it's important to keep the README file clean, professional, and informative. These changes seem to degrade the quality of the documentation. Consider reverting these changes if they were made unintentionally, or replace them with meaningful content that adds value for users and contributors of the project.
```suggestion

```